package.json

```diff
@@ -1,20 +1,26 @@
 {
   "name": "stylelint-config-upleveled",
-  "version": "0.0.1",
-  "description": "",
-  "main": "index.js",
-  "files": [
-    "index.js"
-  ],
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "version": "1.0.1",
+  "description": "UpLeveled Stylelint defaults for styling in CSS, SCSS, Sass, CSS Modules and CSS-in-JS.",
   "keywords": [
     "stylelint",
-    "stylelint-config"
+    "stylelint-config",
+    "config",
+    "css",
+    "sass",
+    "scss",
+    "react",
+    "nextjs"
+  ],
+  "repository": "upleveled/stylelint-config-upleveled",
+  "license": "MIT",
+  "author": "Karl Horky <karl.horky@gmail.com>",
```

Comment:
let's switch this to `author`: UpLeveled (however we do it in the monorepo) and the `contributors`

Reply:
ee76ea3